custom_components/unifiprotect/entity.py

```diff
@@ -45,26 +46,33 @@ def __init__(
             sw_version=self.device.firmware_version,
             connections={(dr.CONNECTION_NETWORK_MAC, self.device.mac)},
             configuration_url=self.device.protect_url,
+            default_manufacturer=DEFAULT_BRAND,
+            default_model="",
+            entry_type=None,
+            identifiers=set(),
+            suggested_area=None,
+            default_name="",
```

Comment:
These should have defaults and shouldn't be needed

Reply:
mypy was complaining about them.

Reply:
```
custom_components/unifiprotect/entity.py:41: error: Missing keys ("default_manufacturer", "default_model", "default_name", "entry_type", "identifiers", "suggested_area") for TypedDict "DeviceInfo"  [typeddict-item]
```